azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -999,6 +999,16 @@ private void ajaxExecuteFlow(final HttpServletRequest req,
     }
     options.setMailCreator(flow.getMailCreator());
 
+    /**
+     * If the user has not explicitly overridden the failure action from the UI or
+     * through API, we will consider the value specified in DSL/Yaml
```

Comment:
Is there a way to specify failure-action/failure-option in 'API'?

Reply:
Yes. API has a way to specifiy failure option with a key `failureAction`. This comes from UI or API.